external_deps/naclsdk-pydir-python3.patch

```diff
@@ -0,0 +1,718 @@
+diff --git a/artools.py b/artools.py
+index 6fa6785..abc4c4a 100755
+--- a/artools.py
++++ b/artools.py
+@@ -1,4 +1,4 @@
+-#!/usr/bin/python
++#!/usr/bin/env python3
```

Comment:
I'm asking to get rid of all these shebang changes. On *nix we always set PNACLPYTHON forcing a specific interpreter when invoking from CMake, so they have no effect.

Reply:
Done.